cmake-script

```diff
@@ -0,0 +1,100 @@
+EXTRA_ARGS=$@
+
+# # To checkout trilinos:
+# # % eg clone software.sandia.gov:/space/git/Trilinos <your-destination-dir>
+
+# # --------
+
+# # To build trilinos (the stk package, for example):
+
+# # % module load compilers/gcc/gcc-4.1.2
+# # % module load openmpi-1.4/gcc-4.1.2
```

Comment:
OpenMPI calls 1.4 "ancient."  The earliest version of OpenMPI that gets regular Trilinos testing (and that works; 1.8 does not) is 1.10.1.